tests/integration/test_properties.py

```diff
@@ -67,6 +67,43 @@ def test_archive_property_by_id(client, auth_headers, new_property, test_databas
     assert responseBadPropertyID.get_json() == {'message': 'Property cannot be archived'}
     assert responseBadPropertyID.status_code == 400
 
+def test_archive_property_by_id(client, auth_headers, new_property, test_database):
```

Comment:
I'd prefer to use the valid_header and empty_test_db factory for all new tests. This will help keep the tests running as fast as possible and prevent issues with tests failing to due changes in the seed file. It doesn't look like the new_property factory is being used here. However, there is a property factory that can be used with the empty test db (it probably needs to be updated to use faker). `tests/factory_fixtures/property.py`
```suggestion
def test_archive_property_by_id(client, valid_header, create_property, empty_test_db):
```

Reply:
I'm not sure I've done the most efficient job of this, but the tests run now and they use the new fixtures `admin_header`, `empty_test_db` and `create_property`
Also, I added faker to the create_property factory
@NickSchimek - you are welcome to check it out!